src/util/db/sqlite.cpp

```diff
@@ -0,0 +1,41 @@
+#include "util/db/sqlite.h"
+
+#include "util/assert.h"
+
+namespace {
+
+/// Date/time formate generated by SQLite CURRENT_TIMESTAMP (UTC) that is
+/// used for library.last_played_at and PlalistTracks.pl_datetime_added.
+const QString kGeneratedTimestampFormat = QStringLiteral("yyyy-MM-dd hh:mm:ss");
+const QString kGeneratedTimestampDateFormat = QStringLiteral("yyyy-MM-dd");
+const QString kGeneratedTimestampTimeFormat = QStringLiteral("hh:mm:ss");
+
+} // namespace
+
+namespace sqlite {
+
+QDateTime readGeneratedTimestamp(const QVariant& value) {
+    const auto timestamp = value.toString();
+    if (timestamp.isEmpty()) {
+        return QDateTime();
+    }
+    const auto parts = timestamp.split(QChar(' '));
```

Comment:
Can this also become a const defineition?

Reply:
No, I will not introduce another constant for this single char.